1-JavaScript/Week3/homworkSolition/ex3-lemonAllergy.test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const fruitBasket = [
+  'apple',
+  'lemon',
+  'grapefruit',
+  'lemon',
+  'banana',
+  'watermelon',
+  'lemon',
+];
+
+function sanitizeFruitBasket(fruitBasket,fruit) {
+
+  const newFruitBasket = [...fruitBasket]
+  const sanitizedBasket = newFruitBasket.filter((fruit)=>{return fruit !== "lemon" })
+  return sanitizedBasket
+}
```

Comment:
Great work! 😄 Try to get used to "implicit returns" for arrow functions, so it's a bit easier to read. Also, you hardcoded "lemon" here, but there is another parameter/argument that you can use in the function" `fruit`. So you can check if that fruit is not equal to the fruit you're are filtering over. Maybe good to change the name too.

```suggestion
function sanitizeFruitBasket(fruitBasket,allergy) {
  const newFruitBasket = [...fruitBasket]
  const sanitizedBasket = newFruitBasket.filter((fruit)=> fruit !== allergy)
  return sanitizedBasket
}
```